pkg/pubsub/pubsub_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * MinIO Cloud Storage, (C) 2019 MinIO, Inc.
```

Comment:
Add a space after the headline
```suggestion
 * MinIO Cloud Storage, (C) 2019 MinIO, Inc.
 
```